src/Amazon.php

```diff
@@ -7,9 +7,16 @@
  */
 class Amazon {
 
+  /**
+   * First Available Shipping Method or null.
+   *
+   * @var WC_Shipping_Rate|null
+   */
+  public static $firstAvailableMethod = NULL;
```

Comment:
If multiple orders are imported in one process, then the same shipping method would be applied to all of them. This should be keyed by order ID.
```suggestion
   * @var WC_Shipping_Rate[]
   */
  public static $firstAvailableMethod = [];
```